Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,6 +454,46 @@ class CredentialsManagerSpec: QuickSpec {
                         }
                     }
                 }
+                
+                it("should yield new credentials when credentials expire in the future") {
```

Comment:
The tests should ideally be testing for 1 case, in this case it doesn't really matter if the credentials have expired or not, as you are manually forcing a refresh. The signal here should be does the renew method on its own return new credentials.